kernel/aster-nix/src/fs/ext2/inode.rs

```diff
@@ -1701,15 +1709,16 @@ impl TryFrom<RawInode> for InodeDesc {
 
 impl InodeDesc {
     pub fn new(type_: FileType, perm: FilePerm) -> Dirty<Self> {
+        let now = now_as_duration(&crate::time::ClockID::CLOCK_REALTIME_COARSE).unwrap();
         Dirty::new_dirty(Self {
             type_,
             perm,
             uid: 0,
             gid: 0,
             size: 0,
-            atime: Duration::ZERO,
-            ctime: Duration::ZERO,
-            mtime: Duration::ZERO,
```

Comment:
@liqinggd I modify this to pass syscall test. `Duration::ZERO` will make `touch` under `ext2` generate files with timestamp 0. I need your suggestions if there are better ways.

Reply:
Get it!